generators/client/templates/angular/src/main/webapp/app/admin/metrics/metrics-modal-threads/metrics-modal-threads.component.html.ejs

```diff
@@ -17,85 +17,85 @@
+      <fa-icon *ngIf="threadStateFilter === undefined" icon="check"></fa-icon>
+      All&nbsp;<span class="badge badge-pill badge-default">{{ threadDumpAll }}</span>
+    </span>
 
-        <span class="badge badge-success hand" (click)="threadStateFilter = ThreadState.Runnable">
-            <fa-icon *ngIf="threadStateFilter === ThreadState.Runnable" icon="check"></fa-icon>
-            Runnable&nbsp;<span class="badge badge-pill badge-default">{{ threadDumpRunnable }}</span>
-        </span>
+    <span class="badge badge-success hand" (click)="threadStateFilter = ThreadState.Runnable">
+      <fa-icon *ngIf="threadStateFilter === ThreadState.Runnable" icon="check"></fa-icon>
+      Runnable&nbsp;<span class="badge badge-pill badge-default">{{ threadDumpRunnable }}</span>
+    </span>
 
-        <span class="badge badge-info hand" (click)="threadStateFilter = ThreadState.Waiting">
-            <fa-icon *ngIf="threadStateFilter === ThreadState.Waiting" icon="check"></fa-icon>
-            Waiting&nbsp;<span class="badge badge-pill badge-default">{{ threadDumpWaiting }}</span>
-        </span>
+    <span class="badge badge-info hand" (click)="threadStateFilter = ThreadState.Waiting">
+      <fa-icon *ngIf="threadStateFilter === ThreadState.Waiting" icon="check"></fa-icon>
+      Waiting&nbsp;<span class="badge badge-pill badge-default">{{ threadDumpWaiting }}</span>
+    </span>
 
-        <span class="badge badge-warning hand" (click)="threadStateFilter = ThreadState.TimedWaiting">
-            <fa-icon *ngIf="threadStateFilter === ThreadState.TimedWaiting" icon="check"></fa-icon>
-            Timed Waiting&nbsp;<span class="badge badge-pill badge-default">{{ threadDumpTimedWaiting }}</span>
-        </span>
+    <span class="badge badge-warning hand" (click)="threadStateFilter = ThreadState.TimedWaiting">
+      <fa-icon *ngIf="threadStateFilter === ThreadState.TimedWaiting" icon="check"></fa-icon>
+      Timed Waiting&nbsp;<span class="badge badge-pill badge-default">{{ threadDumpTimedWaiting }}</span>
+    </span>
 
-        <span class="badge badge-danger hand" (click)="threadStateFilter = ThreadState.Blocked">
-            <fa-icon *ngIf="threadStateFilter === ThreadState.Blocked" icon="check"></fa-icon>
-            Blocked&nbsp;<span class="badge badge-pill badge-default">{{ threadDumpBlocked }}</span>
-        </span>
-    </div>
+    <span class="badge badge-danger hand" (click)="threadStateFilter = ThreadState.Blocked">
+      <fa-icon *ngIf="threadStateFilter === ThreadState.Blocked" icon="check"></fa-icon>
+      Blocked&nbsp;<span class="badge badge-pill badge-default">{{ threadDumpBlocked }}</span>
+    </span>
+  </div>
 
-    <div class="pad" *ngFor="let entry of getThreads() | keyvalue">
-        <h6>
-            <span class="badge" [ngClass]="getBadgeClass(entry.value.threadState)">{{ entry.value.threadState }}</span>
+  <div class="pad" *ngFor="let entry of getThreads()">
```

Comment:
Can you rename variable `entry` to `thread`, because now this is not `Object` entry but `Thread` object.

Reply:
done